### DISCUSSION TITLE
Aros Vision for Amiberry

### DISCUSSION THREAD

**Discussion by amyren1966** [BlitterStudio/amiberry#884] [General]
There is a new version of Aros for 68k. Just tested it, and it seem to run well with Amiberry.
There is a hdf and a directory archive version to download at http://www.aros-platform.de/download.html
I dont know if the new rom files are required, but I downloaded them as well for my test.
If needed, I could post my config file here.

**Comment by midwan** [comment]
Thanks for the information! Previous versions also worked well, as I recall.

**Comment by amyren1966** [comment]
Yes, you are right, the other version works well ok.
This is a different maintainer and is set up to use Dopus5 as the desktop environment.

For the other with Wanderer desktop the maintainer is working on ver 1.3. But this is not yet published.
Homepage is https://sites.google.com/view/arosone
It looks like its only version 1.2 that have a download link: https://ae.amigalife.org/index.php?topic=530.0

**Comment by solskogen** [comment]
Aros for 68k works-ish. Amiberry crashes with JIT on.

**Comment by amyren1966** [comment]
Aros Vision does work here with 020 JIT, FPU JIT, on Manjaro.
I have deleted the setup for the other Aros One 68k disto, but I think I used JIT for that as well.

Even got MUI 4 running on it, and was able to run Hollywood Designer 5 on it. Just for testing, I found it a bit slow on Rpi.

**Comment by solskogen** [comment]
Strange, I can only get it to start without JIT. With JIT on, it will just crash amiberry.

**Reply by amyren1966** [reply]
I found that sometimes if things doesnt work, it will not work even after quitting and relaunching Amiberry. But then works again after restarting my Pi. And then load the config you want to test and run it before running any other config.

I'll post my uae config here if it helps:
`config_description=ArosVision
config_hardware=true
config_host=true
config_version=4.2.0
config_hardware_path=
config_host_path=
config_all_path=
amiberry.hide_idle_led=0
joyport1_name=JOY0
joyport1_friendlyname=MOSIC      SPEED-LINK Competition Pro
joyport1_autofire=none
joyport0_name=MOUSE0
joyport0_friendlyname=Mouse
joyport0_autofire=none
amiberry.rom_path=./
amiberry.floppy_path=./
amiberry.hardfile_path=./
amiberry.cd_path=./
; host-specific
amiberry.middle_mouse=true
amiberry.gfx_auto_height=true
amiberry.expansion_gui_page=ide_mb
; common
use_gui=no
kickstart_rom_file=/home/pi/amiberry/kickstarts/aros-rom.bin
kickstart_ext_rom_file=/home/pi/amiberry/kickstarts/aros-ext.bin
ide_mb_rom_file=:ENABLED
flash_file=
cart_file=
rtc_file=
kickshifter=false
floppy_volume=33
floppy0=
floppy1=
floppy1type=-1
floppy2=
floppy3=
nr_floppies=1
floppy_speed=100
cd_speed=100
parallel_on_demand=false
serial_on_demand=false
serial_hardware_ctsrts=true
serial_direct=false
scsi=true
uaeserial=false
sana2=false
sound_output=exact
sound_channels=stereo
sound_stereo_separation=7
sound_stereo_mixing_delay=0
sound_max_buff=8192
sound_frequency=44100
sound_interpol=none
sound_filter=off
sound_filter_type=standard
sound_volume=0
sound_volume_paula=0
sound_volume_cd=20
sound_volume_ahi=0
sound_volume_midi=0
sound_volume_genlock=0
sound_auto=true
sound_cdaudio=false
sound_stereo_swap_paula=false
sound_stereo_swap_ahi=false
comp_trustbyte=direct
comp_trustword=direct
comp_trustlong=direct
comp_trustnaddr=direct
comp_nf=true
comp_constjump=true
comp_flushmode=soft
compfpu=true
comp_catchfault=true
cachesize=16384
joyport0=mouse
joyport0autofire=none
joyportfriendlyname0=System mouse
joyportname0=MOUSE0
joyport1=joy0
joyport1autofire=none
joyportfriendlyname1=Competition Pro
joyportname1=JOY0
bsdsocket_emu=true
synchronize_clock=false
maprom=0x0
parallel_postscript_emulation=false
parallel_postscript_detection=false
ghostscript_parameters=
parallel_autoflush=5
gfx_display=0
gfx_display_rtg=0
gfx_framerate=1
gfx_width=720
gfx_height=512
gfx_top_windowed=0
gfx_left_windowed=0
gfx_width_windowed=720
gfx_height_windowed=512
gfx_width_fullscreen=1920
gfx_height_fullscreen=1080
gfx_refreshrate=50
gfx_refreshrate_rtg=50
gfx_autoresolution=0
gfx_autoresolution_vga=true
gfx_backbuffers=2
gfx_backbuffers_rtg=1
gfx_vsync=false
gfx_vsyncmode=normal
gfx_vsync_picasso=false
gfx_vsyncmode_picasso=normal
gfx_lores=false
gfx_resolution=hires
gfx_lores_mode=normal
gfx_flickerfixer=false
gfx_linemode=none
gfx_fullscreen_amiga=true
gfx_fullscreen_picasso=true
gfx_center_horizontal=none
gfx_center_vertical=none
gfx_colour_mode=32bit
gfx_blacker_than_black=false
gfx_api=direct3d11
gfx_api_options=hardware
immediate_blits=false
fast_copper=false
multithreaded_drawing=true
ntsc=false
genlock=false
chipset=aga
chipset_refreshrate=49.920410
collision_level=playfields
chipset_compatible=A4000
ksmirror_e0=false
fatgary=0
ramsey=15
z3_autoconfig=true
unmapped_address_space=zero
ide=a4000
z3mapping=real
fastmem_size=0
mem25bit_size=0
a3000mem_size=0
mbresmem_size=0
z3mem_size=256
z3mem_start=0x40000000
bogomem_size=0
gfxcard_hardware_vblank=false
gfxcard_hardware_sprite=false
gfxcard_multithread=false
gfxcard_size=128
gfxcard_type=ZorroIII
chipmem_size=16
cpu_speed=max
cpu_throttle=0.0
cpu_type=68020/68881
cpu_model=68020
fpu_model=68881
cpu_compatible=false
cpu_24bit_addressing=false
cpu_data_cache=false
cpu_cycle_exact=false
cpu_memory_cycle_exact=false
blitter_cycle_exact=false
cycle_exact=false
fpu_strict=false
rtg_nocustom=true
rtg_modes=0x112
kbd_lang=us
hardfile2=rw,DH0:/home/pi/amiberry/Hard-drives_HDF/AROS_HDF2.hdf,0,0,0,512,0,,uae0
uaehf0=hdf,rw,DH0:/home/pi/amiberry/Hard-drives_HDF/AROS_HDF2.hdf,0,0,0,512,0,,uae0
filesystem2=rw,DH1:Extra:/home/pi/amiberry/Hard-drives_HDF/Extra,-128
uaehf1=dir,rw,DH1:Extra:/home/pi/amiberry/Hard-drives_HDF/Extra,-128
input.config=0
input.joymouse_speed_analog=2
input.joymouse_speed_digital=10
input.joymouse_deadzone=33
input.joystick_deadzone=33
input.analog_joystick_multiplier=15
input.analog_joystick_offset=-1
input.mouse_speed=100
input.autofire_speed=0
input.autoswitch=1
input.1.joystick.0.friendlyname=Competition Pro
input.1.joystick.0.name=JOY0
input.1.joystick.0.empty=true
input.1.joystick.1.friendlyname=Competition Pro
input.1.joystick.1.name=JOY1
input.1.joystick.1.empty=true
input.1.joystick.2.empty=true
input.1.joystick.3.empty=true
input.1.joystick.4.empty=true
input.1.joystick.5.empty=true
input.1.joystick.6.empty=true
input.1.joystick.7.empty=true
input.1.mouse.0.friendlyname=System mouse
input.1.mouse.0.name=MOUSE0
input.1.mouse.0.empty=true
input.1.mouse.1.empty=true
input.1.mouse.2.empty=true
input.1.mouse.3.empty=true
input.1.mouse.4.empty=true
input.1.mouse.5.empty=true
input.1.mouse.6.empty=true
input.1.mouse.7.empty=true
input.1.keyboard.0.friendlyname=Default Keyboard
input.1.keyboard.0.name=KEYBOARD0
input.1.keyboard.0.empty=false
input.1.keyboard.0.disabled=false
input.1.keyboard.1.empty=true
input.1.keyboard.2.empty=true
input.1.keyboard.3.empty=true
input.1.keyboard.4.empty=true
input.1.keyboard.5.empty=true
input.1.keyboard.6.empty=true
input.1.keyboard.7.empty=true
input.1.internal.0.friendlyname=Internal events
input.1.internal.0.name=INTERNALEVENTS1
input.1.internal.0.empty=true
input.1.internal.0.disabled=false
input.2.joystick.0.friendlyname=Competition Pro
input.2.joystick.0.name=JOY0
input.2.joystick.0.empty=true
input.2.joystick.1.friendlyname=Competition Pro
input.2.joystick.1.name=JOY1
input.2.joystick.1.empty=true
input.2.joystick.2.empty=true
input.2.joystick.3.empty=true
input.2.joystick.4.empty=true
input.2.joystick.5.empty=true
input.2.joystick.6.empty=true
input.2.joystick.7.empty=true
input.2.mouse.0.friendlyname=System mouse
input.2.mouse.0.name=MOUSE0
input.2.mouse.0.empty=true
input.2.mouse.1.empty=true
input.2.mouse.2.empty=true
input.2.mouse.3.empty=true
input.2.mouse.4.empty=true
input.2.mouse.5.empty=true
input.2.mouse.6.empty=true
input.2.mouse.7.empty=true
input.2.keyboard.0.friendlyname=Default Keyboard
input.2.keyboard.0.name=KEYBOARD0
input.2.keyboard.0.empty=false
input.2.keyboard.0.disabled=false
input.2.keyboard.1.empty=true
input.2.keyboard.2.empty=true
input.2.keyboard.3.empty=true
input.2.keyboard.4.empty=true
input.2.keyboard.5.empty=true
input.2.keyboard.6.empty=true
input.2.keyboard.7.empty=true
input.2.internal.0.friendlyname=Internal events
input.2.internal.0.name=INTERNALEVENTS1
input.2.internal.0.empty=true
input.3.joystick.0.friendlyname=Competition Pro
input.3.joystick.0.name=JOY0
input.3.joystick.0.empty=true
input.3.joystick.1.friendlyname=Competition Pro
input.3.joystick.1.name=JOY1
input.3.joystick.1.empty=true
input.3.joystick.2.empty=true
input.3.joystick.3.empty=true
input.3.joystick.4.empty=true
input.3.joystick.5.empty=true
input.3.joystick.6.empty=true
input.3.joystick.7.empty=true
input.3.mouse.0.friendlyname=System mouse
input.3.mouse.0.name=MOUSE0
input.3.mouse.0.empty=true
input.3.mouse.1.empty=true
input.3.mouse.2.empty=true
input.3.mouse.3.empty=true
input.3.mouse.4.empty=true
input.3.mouse.5.empty=true
input.3.mouse.6.empty=true
input.3.mouse.7.empty=true
input.3.keyboard.0.friendlyname=Default Keyboard
input.3.keyboard.0.name=KEYBOARD0
input.3.keyboard.0.empty=false
input.3.keyboard.0.disabled=false
input.3.keyboard.1.empty=true
input.3.keyboard.2.empty=true
input.3.keyboard.3.empty=true
input.3.keyboard.4.empty=true
input.3.keyboard.5.empty=true
input.3.keyboard.6.empty=true
input.3.keyboard.7.empty=true
input.3.internal.0.friendlyname=Internal events
input.3.internal.0.name=INTERNALEVENTS1
input.3.internal.0.empty=true
input.4.joystick.0.friendlyname=Competition Pro
input.4.joystick.0.name=JOY0
input.4.joystick.0.custom=true
input.4.mouse.0.friendlyname=System mouse
input.4.mouse.0.name=MOUSE0
input.4.mouse.0.custom=true
input.4.keyboard.0.friendlyname=Default Keyboard
input.4.keyboard.0.name=KEYBOARD0
input.4.keyboard.0.custom=true
; *** WHDLoad Booter. Options
whdload_slave=
whdload_showsplash=true
whdload_buttonwait=false
whdload_custom1=0
whdload_custom2=0
whdload_custom3=0
whdload_custom4=0
whdload_custom5=0
whdload_custom=
`

**Comment by solskogen** [comment]
Are you running Xorg perhaps? This might be one of those things where amiberry crashes in KMSDRM mode, but not when running in Xorg.

**Comment by solskogen** [comment]
No crash with JIT in Xorg :)

**Reply by amyren1966** [reply]
I am using Manjaro 64bit, KDE Plasma version 5.20.5, KDE Frameworks version 5.78.0, core version 5.4.83-1-MANJARO-ARM
